src/TraceEvent/Microsoft.Diagnostics.Tracing.TraceEvent.targets

```diff
@@ -0,0 +1,37 @@
+<!--
+***********************************************************************************************
+Microsoft.Bcl.Compression.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
```

Comment:
➡️ I copied this from the existing TraceEvent NuGet package without modification.